selenium_tests/management/commands/snapshot_dashboard_states.py

```diff
@@ -299,7 +297,7 @@ def test_dashboard_states(self):
             scenarios_with_numbers = (
                 (num, (run_scenario, name))
                 for (num, (run_scenario, name)) in scenarios_with_numbers
-                if match in self._make_filename(num, name)
+                if match in self._make_filename(num, name, output_directory)
```

Comment:
We might want to pass an empty string for the output directory here, so that we don't match against the directory name by accident